docs/Features/UI.md

```diff
@@ -45,6 +45,17 @@ Accessible through <b>Options -> Epip settings -> General -> Show Aggro Informat
 <tr><td>Taunted:     </td><td><img src="../img/ui/aggro_taunted.png"/></td></tr>
 </table>
 </details>
+<details>
+<summary>Aggro tags explanation</summary>
+AI chooses its targets based on a number of parameters, such as target's effective HP, how easy it is to get to the target and so on. Its behavior is also influenced by aggro tags.
```

Comment:
Worth mentioning that the explanations of these tags are valid for EE only. They do not work the same in vanilla (no need to explain that though)